### DISCUSSION TITLE
Free seed chest not effective against bots

### DISCUSSION THREAD

**Discussion by Polysemouse** [sunflower-land/sunflower-land#1388] [General]
**Problem**
Currently we added small noise color to the free seed chest (No. 9 In [#481](https://github.com/sunflower-land/sunflower-land/issues/481)):

> make coloured chest and add mini noise to image, because Neural Network easy detect imaged items with no color difference and noise.

The way it's implemented now is ineffective, because the bumpkin (which has no color noise) always appear at an equal distance away from the center from the screen relative to the chest.  The mentioned Neural Networks can simply recognize the bumpkin and click the opposite side of the screen to open the chest.

**Suggestions**
Either randomize the location of the bumpkin a bit more, and/or add some small noise to the bumpkin

![Screenshot_20220901-124256](https://user-images.githubusercontent.com/107602352/188199198-bdd618c8-8f1c-41e5-9661-6f38f3eb2993.png)

**Comment by adamhannigan** [comment]
@Polysemouse Great find, we didn't think about that.

Are you interested in trying to implement this solution or do you want a developer to look into it?

**Reply by Polysemouse** [reply]
@adamhannigan  I wish I could learn to code for the front end, it would be so cool.  I would want some developers to look into it.  Thanks for taking look at the issue!

**Comment by hujikol** [comment]
I have some idea to disturb the image with neural network too, but i think it's too overpowered in case of this scenario.

in case you want to check it out https://youtu.be/AOZw1tgD8dA

**Comment by Polysemouse** [comment]
I am assuming that https://github.com/sunflower-land/sunflower-land/pull/1437 has fixed the issue
